tools/js-sdk-release-tools/src/rlcCodegenCli.ts

```diff
@@ -32,38 +32,35 @@ async function autoGenerate(options: any) {
         packageName = options['package-name'];
         if (!packageName || !validPackageName(packageName)) {
             if (!!packageName && !validPackageName(packageName)) {
-                logger.warn(`Your package-name ${packageName} is invalid, it should be in format @azure-rest/xxxxx.`)
+                logger.warn(`Package name '${packageName}' is invalid, it should be in format '@azure-rest/xxxxx'.`)
             }
             packageName = await getPackageNameFromCommand();
         }
         packagePath = findPackageInRepo(packageName, sdkRepo);
         if (!packagePath) {
-            logger.info(`${packageName} is first generated.`);
+            logger.info(`'${packageName}' is first generated.`);
             const rp = options['service-name']? options['service-name'] : await getInputFromCommand('service-name');
             createFolderIfNotExist(path.join(sdkRepo, 'sdk', rp));
             createFolderIfNotExist(path.join(sdkRepo, 'sdk', rp, getPackageFolderName(packageName)));
             packagePath = path.join(sdkRepo, 'sdk', rp, getPackageFolderName(packageName));
             await generateSampleReadmeMd(packageName, packagePath, options);
         } else {
             if (!fs.existsSync(path.join(packagePath, 'swagger', 'README.md'))) {
-                logger.info(`${packageName} is found in ${packagePath}, but not contains swagger/README.md. Creating a sample one for quickstart.`);
+                logger.info(`'${packageName}' is found in ${packagePath}, but swagger/README.md is not found. Start to create a sample one for quickstart.`);
                 await generateSampleReadmeMd(packageName, packagePath, options);
             } else if (options['interactive']) {
                 await modifyExistingReadmeMd(packageName, packagePath);
             }
         }
     }
     if (!packageName) {
-        logger.error(`Cannot get valid package-name.`);
+        logger.error(`Package name '${packageName}' is invalid.`);
```

Comment:
Are there any next steps for the user?

Reply:
It's not design for external users, you can ignore it